profiling/src/profiling/mod.rs

```diff
@@ -658,6 +664,54 @@ impl Profiler {
         }
     }
 
+    #[cfg(feature = "timeline")]
+    pub unsafe fn collect_timeline_gc_event(
+        &self,
+        duration: i64,
+        reason: &str,
+        locals: &RequestLocals,
+    ) {
+        lazy_static! {
+            static ref TIMELINE_GC_LABELS: Vec<Label> = vec![
+                Label {
+                    key: "event",
+                    value: LabelValue::Str("gc".into()),
+                },
+                Label {
+                    key: "thread id",
+                    value: LabelValue::Str("GC".into()),
+                },
```

Comment:
I am confused. The labels "event: gc" and "thread id: GC" mean... what?

Reply:
- the `thread id` makes it go into the extra "virtual" garbage collection thread in the visualisation
- the `gc` in `event` marks it a garbage collection event so it will be displayed as such in the frontend

Our alternatives (and I would have to play with this) would be to not set a `thread id` and have this GC-event in the "main" thread, which would be more correct in the case of PHP, but IDK if the backend supports this for visualisation, but I think I will try this out.

Reply:
I removed the `thread id`, as in the current version of the backend `gc` type events will be shown in this virtual gc thread anyway. We can solve this part on the backend then.